sympy/algebras/quaternion.py

```diff
@@ -1112,8 +1114,8 @@ def index_vector(self):
         Explanation
         ===========
 
-        Index vector is given by $mod(q)$ multiplied by $\mathbf{Ax}(q)$ where Ax(q) is the axis of the quaternion q,
-        and mod(q) is the `norm` (magnitude) of the quaternion.
+        Index vector is given by $\mathbf{T}(q)$ multiplied by $\mathbf{Ax}(q)$ where $Ax(q)$ is the axis of the quaternion q,
+        and mod(q) is the $norm$ (magnitude) of the quaternion.
```

Comment:
Please just replace `mod(q)` by `$\mathbf{T}(q)$` here. It would be nice to repeat the `$\mathbf{Ax}(q)$` instead of writing it without the `\mathbf`.